testsuite/tests/lib-threads/mutex_errorcheck.reference

```diff
@@ -0,0 +1,3 @@
+start
+Sys_error("Mutex.lock: Resource deadlock avoided")
```

Comment:
The exact string reported here is libc- and locale-dependent. This test shouldn't fail if it changes.